modules/extension/wms/src/main/java/org/geotools/map/WMSCoverageReader.java

```diff
@@ -112,6 +113,8 @@
      * Last request CRS (used for reprojected GetFeatureInfo)
      */
     CoordinateReferenceSystem requestCRS;
+
+    public List<String> styles = new ArrayList<>();
```

Comment:
Public? I would also put it side by side with layers for the sake of clarity (parallel lists).